browser/resources/brave_new_tab_page_refresh/state/background_state.ts

```diff
@@ -132,22 +147,19 @@ export function getCurrentBackground(
 
   switch (type) {
     case SelectedBackgroundType.kBrave: {
-      if (currentBackground?.type === 'brave') {
-        return currentBackground
-      }
-      const braveBackground = chooseRandom(braveBackgrounds)
+      const braveBackground = chooseIndex(braveBackgrounds, rotateIndex)
       return braveBackground ? { type: 'brave', ...braveBackground } : null
     }
     case SelectedBackgroundType.kCustom: {
-      const imageUrl = value || chooseRandom(customBackgrounds)
+      const imageUrl = value || chooseIndex(customBackgrounds, rotateIndex)
```

Comment:
fwiw, I'm pretty sure the existing NTP uses random, rather than rotating through - not saying we shouldn't do this just that its different